src/ontology/mondo-ingest.Makefile

```diff
@@ -509,6 +506,40 @@ slurp-all-no-updates: $(foreach n,$(ALL_COMPONENT_IDS), slurp-no-updates-$(n))
 .PHONY: slurp-all
 slurp-all: $(foreach n,$(ALL_COMPONENT_IDS), slurp-$(n))
 
+
+#############################
+###### Synchronization ######
+#############################
+.PHONY: generate-synchronization-files
+generate-synchronization-files: sync-subclassof
+
+.PHONY: sync-subclassof
+sync-subclassof: $(REPORTDIR)/sync-subClassOf.in-mondo-only.tsv
+
+.PHONY: sync-subclassof-%
+sync-subclassof-%:
+	$(MAKE) $(REPORTDIR)/$*.subclass.added.robot.tsv
+
+# Side effects: Deletes SOURCE.subclass.in-mondo-only.tsv's from which the combination is made.
+$(REPORTDIR)/sync-subClassOf.in-mondo-only.tsv: $(foreach n,$(ALL_COMPONENT_IDS), sync-subclassof-$(n))
+	python3 $(SCRIPTSDIR)/sync_subclassof.py --collate-in-mondo-only $@
+
+$(REPORTDIR)/sync-subClassOf.confirmed.tsv: $(foreach n,$(ALL_COMPONENT_IDS), $(REPORTDIR)/$(n).subclass.confirmed.robot.tsv)
```

Comment:
## `reports/sync-subClassOf.confirmed.tsv` @matentzn
@matentzn Questions
<details><summary>Details</summary>
<p>

@matentzn For most of these changes / improvements, I probably would have to change implementation though from `awk` to Python FYI.

### 1. Column order
Order is currently: `subject_mondo_id, subject_mondo_label, object_mondo_id, subject_source_id, object_source_id, object_mondo_label`
But would having these 4 columns on the left make more sense, as they are essentially the composite primary key of this combined file: `subject_mondo_id, object_mondo_id, subject_source_id, object_source_id`

### 2. Additional `source` column?

### 3. Sorting
Right now they are sorted by source since they are concatenated together. I think it only makes sense to change the sorting if we added the `source` column. Then, I would sort by: `source, mondo_subject_id, mondo_object_id`

### 4. File name
I now have combined files that have the stem `subClassOf`, e.g. `sync-subClassOf.confirmed.tsv` and `sync-subClassOf.in-mondo-only.tsv`, but the non-combined files have the stem `subclass`, e.g. `*.subclass.confirmed.robot.tsv`. Should I change so that the stems are consistently 1 or the other (`subclass` or `subClassOf`)?

</p>
</details>

Reply:
TMD for me - feel free to make a call!

Reply:
Alright, opting against for all of these in interest of time. But I added a lowercase `todo` (lower priority) to the top of `sync_subclassof.py` to remind of future improvements.